src/assets/js/themes.js

```diff
@@ -17,27 +19,48 @@
         var switcher = document.getElementById('js-theme-switcher');
         switcher.removeAttribute('hidden');
         var light_theme_toggle = document.getElementById('light-theme-toggle'),
-            dark_theme_toggle = document.getElementById('dark-theme-toggle');
+            dark_theme_toggle = document.getElementById('dark-theme-toggle'),
+            system_theme_toggle = document.getElementById('system-theme-toggle');
+        var isSystemTheme = document.documentElement.getAttribute("data-system-theme") === "true";
```

Comment:
```suggestion
        var isSystemTheme = document.documentElement.getAttribute("data-system-theme") === "true";
```
We don't need "data-system-theme" attribute as well.